fe/fe-core/src/main/java/com/starrocks/sql/analyzer/WorkGroupAnalyzer.java

```diff
@@ -56,6 +61,17 @@ public static WorkGroupClassifier convertPredicateToClassifier(List<Predicate> p
                 } else if (key.equalsIgnoreCase(WorkGroup.SOURCE_IP)) {
                     SubnetUtils.SubnetInfo subnetInfo = new SubnetUtils(value).getInfo();
                     classifier.setSourceIp(subnetInfo.getCidrSignature());
+                } else if (key.equalsIgnoreCase(WorkGroup.DATABASES)) {
+                    List<String> databases = Splitter.on(",").splitToList(value);
+                    if (CollectionUtils.isEmpty(databases)) {
+                        throw new SemanticException(
+                                String.format("Illegal classifier specifier '%s': '%s'", WorkGroup.DATABASES, eqPred));
+                    }
+                    String clusterName = ConnectContext.get() != null ? ConnectContext.get().getClusterName() : "";
+                    List<String> fullNames = databases.stream()
+                            .map(x -> ClusterNamespace.getFullName(clusterName, x))
+                            .collect(Collectors.toList());
+                    classifier.setDatabases(fullNames);
```

Comment:
for user interface， we could use database name. but I think in our system, we should use database id. then, we needn't currentSqlDbNames in ConnectContext, and  long set performance is better than string set.

Reply:
Good advice. I have changed it.